zephyr/src/supp_main.c

```diff
@@ -404,6 +404,14 @@ static void z_wpas_start(void)
 		goto out;
 	}
 
+#if defined (CONFIG_WPA_SUPP_LOG_LEVEL_DBG) && defined (CONFIG_WPA_SUPP_DEBUG_SHOW_KEYS)
+	wpa_supplicant_set_debug_params(global, MSG_DEBUG, 1, 1);
+#elif defined (CONFIG_WPA_SUPP_LOG_LEVEL_DBG)
+	wpa_supplicant_set_debug_params(global, MSG_DEBUG, 1, 0);
+#else
+	wpa_supplicant_set_debug_params(global, MSG_INFO, 0, 0);
```

Comment:
So for the default we will end up with INFO. In Matter we set `WPA_SUPP_LOG_LEVEL_ERR`=y - is it going to be overridden by the code above?